jboss-kie-kieserver/tests/bats/jboss-kie-kieserver.bats

```diff
@@ -617,3 +617,34 @@ teardown() {
     echo "GC_MAX_METASPACE_SIZE=${GC_MAX_METASPACE_SIZE}"
     [[ "${GC_MAX_METASPACE_SIZE}" == "${KIE_SERVER_MAX_METASPACE_SIZE}" ]]
 }
+
+@test "Verify the Kafka extension" {
+    export KIE_SERVER_KAFKA_EXT_ENABLED="true"
+    export KIE_SERVER_KAFKA_EXT_BOOTSTRAP_SERVERS="localhost:9092"
+    export KIE_SERVER_KAFKA_EXT_CLIENT_ID="app"
+    export KIE_SERVER_KAFKA_EXT_GROUP_ID="jbpm-consumer"
+    export KIE_SERVER_KAFKA_EXT_ACKS="2"
+    export KIE_SERVER_KAFKA_EXT_MAX_BLOCK_MS="2000"
+    export KIE_SERVER_KAFKA_EXT_AUTOCREATE_TOPICS="true"
+    export KIE_SERVER_KAFKA_EXT_TOPICS="person=human,dog=animal,ant="
+
+    configure_kafka
+
+    local expected=" -Dorg.kie.server.jbpm-kafka.ext.disabled=false -Dorg.kie.server.jbpm-kafka.ext.bootstrap.servers=localhost:9092 -Dorg.kie.server.jbpm-kafka.ext.client.id=app -Dorg.kie.server.jbpm-kafka.ext.group.id=jbpm-consumer -Dorg.kie.server.jbpm-kafka.ext.acks=2 -Dorg.kie.server.jbpm-kafka.ext.max.block.ms=2000 -Dorg.kie.server.jbpm-kafka.ext.allow.auto.create.topics=true -Dorg.kie.server.jbpm-kafka.ext.topics.person=human -Dorg.kie.server.jbpm-kafka.ext.topics.dog=animal"
+    echo "  Result: ${JBOSS_KIE_ARGS}"
+
+    echo "Expected: ${expected}"
+    [[ "${JBOSS_KIE_ARGS}" == "${expected}" ]]
```

Comment:
add a check to make sure that "ant=" is not set, like checking the log message that it would have to print.

Reply:
Isn't present in the system properties expected

Reply:
should be added there as well, bats tests should cover all possibilities as well. :)

Reply:
Here: https://github.com/jboss-container-images/jboss-kie-modules/pull/446/files#diff-c6b09ca9c95a186444fb9d39db3c31baee18124133826800ccedc74a82646bf2R652-R668